pages/linux/pvcreate.md

```diff
@@ -1,11 +1,11 @@
 # pvcreate
 
-> Initialize a disk or partition for use by LVM (Logical Volume Manager).
+> Initialize a physical volume (disk or partition) for use by the Logical Volume Manager (LVM).
 
-- Initialize `/dev/sda1` to use lvm:
+- Initialize the `/dev/sda1` device for use by LVM:
```

Comment:
In the main description, the terminology used is "disk or partition", but here you use "device". I think its good to be consistent. Maybe we can use `/dev/sda1` partition here ?